src/entities/csv-info.ts

```diff
@@ -21,6 +21,6 @@ export class CsvInfo extends BaseEntity {
         onDelete: 'CASCADE',
         orphanedRowAction: 'delete'
     })
-    @JoinColumn({ name: 'import_id' })
+    @JoinColumn({ name: 'import_id', foreignKeyConstraintName: 'FK_csv_info_import_id' })
```

Comment:
I've added explicit PK and FK naming because the auto-generated names from typeorm are basically indecipherable once they're created. This makes it a pain to remove or modify them later.

Reply:
Happy with this and letting TypeOrm do the heavy lifting on migration generation.